File: src/main/java/easyhattrickmanager/controller/model/PlayerDataResponse.java
package easyhattrickmanager.controller.model;

import easyhattrickmanager.service.model.dataresponse.UserConfig;
import easyhattrickmanager.service.model.playerdataresponse.PlayerWeeklyInfo;
import java.util.List;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class PlayerDataResponse {

    List<PlayerWeeklyInfo> weeklyData;
    UserConfig userConfig;
}

File: frontend/src/app/services/model/player-data-response.ts
import {PlayerInfo, UserConfig} from './data-response';

export interface PlayerDataResponse {
  weeklyData: PlayerWeeklyInfo[];
  userConfig: UserConfig;
}

export interface PlayerWeeklyInfo {
  season: number;
  week: number;
  date: string;
  player: PlayerInfo;
}
